nmxact/xact/fs.go

```diff
@@ -27,6 +27,11 @@ import (
 	"github.com/recogni/newtmgr/nmxact/sesn"
 )
 
+//////////////////////////////////////////////////////////////////////////////
+
+// Recogni: Limit the max file system based upload chunk size to 1300 bytes.
+const cMaxFSUploadChunk = 1300
```

Comment:
I made this more go-like. It will be good if we can elaborate on why this limit is needed.